src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+define(function () {
+
+  var webauthn = (function () {
+
+    const webauthnDB = (function () {
```

Comment:
How does it work? If I'm in IE10 (which doesn't know about const) and I evaluate this line, why isn't it throwing an error? 

Also, if you wrote this logic - then my suggestion is to not use const or any of these things that are not supported by IE10, etc. We are planning on updating our build to support this, but we need to first integrate a transpiler (i.e. Babel) to convert these types of statements down to es5.